ZUGFeRD/InvoiceDescriptor21Reader.cs

```diff
@@ -579,7 +600,27 @@ private static Party _nodeAsParty(XmlNode baseNode, string xpath, XmlNamespaceMa
                 retval.ContactName = null;
             }
 
+            retval.AddressLine3 = _nodeAsString(node, "ram:PostalTradeAddress/ram:LineThree", nsmgr);
+            retval.CountrySubdivisionName = _nodeAsString(node, "ram:PostalTradeAddress/ram:CountrySubDivisionName", nsmgr);
+
             return retval;
         } // !_nodeAsParty()
+
+        private static AdditionalReferencedDocument GetAdditionalReferencedDocument(XmlNode a_oXmlNode, XmlNamespaceManager a_nsmgr)
```

Comment:
can you please follow the style guide and use prefix _ along with a lower case letter for the function?